custom/download_qrda.php

```diff
@@ -205,15 +205,15 @@ function closeMe() {
                 echo "</td>";
                 echo "<td align=center>";
                 echo "<div id=download" . attr($counter) . ">";
-                echo "<img class='downloadIcon' src='" . $GLOBALS['webroot'] . "/images/downbtn.gif' onclick=downloadXML(" . attr_js($counter) . ",1); />";
+                echo "<img class='downloadIcon' src='" . $GLOBALS['images_static_relative'] . "/downbtn.gif' onclick=downloadXML(" . attr_js($counter) . ",1); />";
                 echo "</div>";
                 echo "<div style='display:none' id=spin" . attr($counter) . ">";
-                echo "<img src='" . $GLOBALS['webroot'] . "/interface/pic/ajax-loader.gif'/>";
+                echo "<img src='" . $GLOBALS['images_static_relative'] . "/interface/pic/ajax-loader.gif'/>";
```

Comment:
this doesn't look right. Shouldn't there just be /ajax-loader.gif rather than /interface/pic/ajax-loader.gif ?

Reply:
The $GLOBALS variable leads to $web_root/public/images, but this gif is in interface/pics, not public/images. Wouldn't that lead to an incorrect path?

Reply:
Then would revert this line back to original:
```
<img src='<?php echo $GLOBALS['webroot'];?>/interface/pic/ajax-loader.gif'/>
```